hub/saml-engine/src/main/java/uk/gov/ida/hub/samlengine/SamlEngineConfiguration.java

```diff
@@ -111,11 +111,11 @@ public SamlConfiguration getSamlConfiguration() {
     }
 
     public Optional<CountryConfiguration> getCountryConfiguration() {
-        return country != null ? Optional.of(country) : Optional.empty();
+        return Optional.ofNullable(country);
     }
 
     public boolean isEidasEnabled() {
-        return getCountryConfiguration().isPresent();
+        return getCountryConfiguration().isPresent() && getCountryConfiguration().get().isEnabled();
```

Comment:
_small innocuous point_ Optional best practice would be something like:

`return getCountryConfiguration().map(CountryConfiguration::isEnabled).orElse(false)`

Reply:
Fair point - however I'm planning on removing the Optional and just doing `getCountryConfiguration.isEnabled` next release :D I figured this was easier to review the new behaviour